motion.c

```diff
@@ -2761,27 +2761,29 @@ int main (int argc, char **argv)
          * option is used, so start at 1 then and 0 otherwise.
          */
         for (i = cnt_list[1] != NULL ? 1 : 0; cnt_list[i]; i++) {
-            /* If i is 0 it means no thread files and we then set the thread number to 1 */
-            cnt_list[i]->threadnr = i ? i : 1;
```

Comment:
If I've understood correctly, there are still other places we use threadnr, so we probably don't want to delete this line? ie. we still want to set threader  (Or we want to update the other places... not sure.)